files/initramfs-tools/arista-convertfs.j2

```diff
@@ -82,6 +82,24 @@ run_cmd() {
     fi
 }
 
+create_varlog_file() {
```

Comment:
the reason I think it should be in the installation process is to save the boot time.